.github/workflows/devcontainer.yml

```
@@ -0,0 +1,60 @@
+
+      - name: Build image
+        uses: docker/build-push-action@v4
+        with:
+          # If the event is a "pull_request", just test the image builds successfully and do not push it
+          push: ${{ github.event_name != 'pull_request' }}
+          context: .devcontainer
+          target: dev
+          build-args: |
+            node_version=${{ steps.node-version.outputs.result }}
+          platforms: linux/amd64
+          tags: ${{ inputs.meta.outputs.tags }}
```

actionlint (1.7.7)

property "meta" is not defined in object type {}

(expression)